index.html

```diff
@@ -70,6 +70,24 @@ <h2>Software Developer</h2>
         <a href="#"><img src="images/arrow.svg" alt="scroll-down-recent-work"></a>
       </div>
     </section>
+
+    <section class="work workbg">
+      <h1>MY RECENT WORK</h1>
+      <img src="images/vector.svg" alt="separation-line">
+      <img src="images/placeholder.svg" alt="image">
+      <h2>Multi-Post Stories</h2>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+      <ul class="workul">
+        <li><a href="#">CSS</a></li>
+        <li><img src="images/divider.svg" alt=""></li>
+        <li><a href="#">HTML</a></li>
+        <li><img src="images/divider.svg" alt=""></li>
+        <li><a href="#">Bootstrap</a></li>
+        <li><img src="images/divider.svg" alt=""></li>
+        <li><a href="#">Ruby</a></li>
```

Comment:
According to the project requirement, Kindly implement the button interactions (enable, hover, etc.)